### ISSUE TITLE
No sound using Adam image

### ISSUE THREAD

**Issue by FandangoInPetto** [eduardofilo/RG350_adam_image#165]
Hi, I just got a new **RG280V**, tested it a bit with stock firmware, everything worked and there was sound. I then flashed Adam image **1.3.1** on it (did everything right including selecting the correct kernel) and everything worked except the sound.
I've tested it on a bunch of different nes/famicom games, nothing.
Oddly enough if I plug headphones into the audio jack and unplug them (the headphones jack itself functions without any problem) the speaker sound is now present and I can control it fine with all the built-in controls (buttons or select + up/down or power + up/down).
Until it goes again after putting the console to sleep or turning it off and then waking/turning it on.

I tried flashing version **1.3.0**, the same problem appears.

I finally came back to stock firmware and everything works fine!

Some observations:

- there's a little popping sound when a game starts (even when the sound works),
- when using Adam alsamixer shows the "speaker" channel as muted ("MM") whereas it's "00" while using stock firmware,
- out of curiosity I took the console apart to see if some connector was faulty but everything, to the best of my knowledge, checks out

Thanks in advance for any input regarding this, Adam image is so much better than the stock firmware!
And thanks for all the efforts and work that went into making and maintaining this!

**Comment by eduardofilo** [comment]
Hi.
It is weird. I have a RG280V and I don't experience those problems.

The popping sound is a classic in this machine. ZCRC explained one time that in this machine, the speaker is connected directly to DAC conversor, and not through the headphones amplifier like in RG350 series. I've always put the noise down to that, but I don't really know.

What surprises me the most is the mute you mention. I see 00 on that channel.

![screenshot001](https://user-images.githubusercontent.com/2923728/149832540-41fd016b-20e1-42d7-b2bd-a448fb88d06a.png)

It occurs to me that the problem may come from the headphone port having a sensor that detects when the jack is punctured and in that situation the system mutes the sound. Surely in your machine that sensor is continuously marking the connected jack. In fact, I just did the test connecting some headphones and indeed now I see the MM thing you were saying. But when I remove them it returns to 00 (I have to exit and enter alsamixer every time so that it refreshes). The stock system shouldn't use that sensor and that's why you have audio.

![screenshot002](https://user-images.githubusercontent.com/2923728/149833090-613762f9-c531-451e-b68e-d1c7c957e548.png)

I don't know if the manufacturer will consider it a problem for warranty claim as the stock system has no problem with this.

Bad luck.

**Comment by eduardofilo** [comment]
Take a closed look at the jack connector soldering. Maybe there is a bridge.

**Comment by FandangoInPetto** [comment]
Hi Eduardo,
Thank you for the quick, detailed reply, I took the console apart again and you were right, there was a bridge on the jack connector!
It definitely explains why I've been having this issue if the stock firmware doesn't take the "jacked-in" sensor into account.

Pic of the bridge:
![IMG_5670](https://user-images.githubusercontent.com/13013575/149894420-b1bd6de9-27ef-4d9b-b0e2-ff0bc80a761c.jpeg)
I scraped it off and now everything works perfectly using Adam image 1.3.1!

Thank you again for the assistance and your hard work!
